assembl/static2/js/app/index.jsx

```diff
@@ -10,17 +10,17 @@ import createAppStore from './store';
 import client from './client';
 import Routes from './routes';
 
-
 /*
 `piwik-react-router` expects that we provide it with a browser history.
 We provide it with `react-router`'s `browserHistory`.
 This is not the official way of doing it.
 The official way is using the `history` library, called with `import history from './utils/history';`.
-For details, see https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components
+For details, see
+https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components
 */
 const history = browserHistory;
 
-const store = createAppStore(client);
+const store = createAppStore();
```

Comment:
There is no real reason to have the apollo cache in the same redux store, except if you use redux-persist with localstorage to have some offline features.
In next version of apollo-client, you can't integrate the apollo cache inside your redux store anyway. They removed the redux dependency.